kibble/render/file_renderer.go

```diff
@@ -96,9 +96,6 @@ func (c FileRenderer) Render(templatePath string, filePath string, data jet.VarM
 	os.MkdirAll(dirPath, 0777)
 
 	// optional check
-	if _, err := os.Stat(fullPath); err == nil {
```

Comment:
Why remove this?  This tells us when duplicate things are causing trouble e.g. bundles.

Reply:
Was just temporary. The warning was printed out at the top of the webpage a bunch I think. Didn't know how else to fix it.